out_writeapi_test.go

```diff
@@ -38,3 +44,143 @@ func TestFLBPluginRegister(t *testing.T) {
 	assert.Equal(t, "writeapi", currplugin.name)
 
 }
+
+type StreamChecks struct {
+	calledGetContext     bool
+	calledcheckResponses bool
+	createDecoder        bool
+	gotRecord            bool
+	calledparseMap       bool
+	calledJTB            bool
+	appendRows           bool
```

Comment:
Change from bool to an integer. That way we can count and verify the number of times this operation is called.

Reply:
Done.